apps/playground/src/app/api/chat/route.ts

```
@@ -106,18 +124,328 @@ export async function POST(req: Request) {
+				const timeoutPromise = new Promise<never>((_, reject) => {
+					setTimeout(
+						() =>
+							reject(new Error(`MCP connection timeout for ${server.name}`)),
+						10000,
+					);
+				});
+
+				const client = await Promise.race([clientPromise, timeoutPromise]);
+				mcpClients.push({ client, name: server.name });
+			} catch (error) {
+				console.error(
```

ESLint

[error] Unexpected console statement.

(no-console)